spec/order_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'order'
+
+describe Order do
+  context 'giving it a menu' do
```

Comment:
I guess you should use "#describe" to describe a method and context to separate the functions of the same method when you are using it in different situations. I can probably explain it better when we make the call.